src/components/Multiselect/types.ts

```diff
@@ -0,0 +1,11 @@
+import { UseComboboxPropGetters } from "downshift7";
+import { ReactNode } from "react";
+
+export type MultiselectOption = { label: string; value: string };
```

Comment:
suggestion: IMO this is a little bit redundant, we should have singe type for example Option